erpnext_documentation/www/docs/user/manual/en/support/issue.md

```diff
@@ -15,7 +15,7 @@ To access the Issue list, go to:
 ## 1. Prerequisites
 Before creating and using Issues, it is advised that you create the following first:
 
-* [Customer](/docs/user/manual/en/CRM/customer)
+* [Customer](/docs/user/manual/en/CRM/customer), with `Disabled` checkbox unselected
```

Comment:
Don't think this needs to be specified here. Its an unspoken rule that it will have to be enabled to use it anywhere

Reply:
agree with this